test/regressions/vite.config.js

```diff
@@ -0,0 +1,49 @@
+import url from 'url';
+import path from 'path';
+import { defineConfig, transformWithEsbuild } from 'vite';
+import react from '@vitejs/plugin-react';
+
+const currentDirectory = url.fileURLToPath(new URL('.', import.meta.url));
+
+export default defineConfig({
+  build: {
+    outDir: 'build',
+  },
+  resolve: {
+    alias: {
+      '@mui/docs': path.resolve(
+        currentDirectory,
+        '../../node_modules/@mui/monorepo/packages/mui-docs/src',
+      ),
+      docsx: path.resolve(currentDirectory, '../../docs'),
+    },
+  },
+  worker: {
+    format: 'es',
+  },
+  plugins: [
+    react(),
+    {
+      name: 'replace-code',
+      enforce: 'post',
+      async transform(code) {
+        return code
+          .replaceAll('__RELEASE_INFO__', 'MTU5NjMxOTIwMDAwMA==') // 2020-08-02
+          .replaceAll('DISABLE_CHANCE_RANDOM', 'true');
+      },
```

Comment:
Nice simplification instead of an explicit plugin. 👍

Reply:
I did it this way because `vite-plugin-replace` wasn't replacing the strings in a node_modules dependency (`x-data-grid-generator`), but it happens to be less verbose and one less dependency so I'm happy with this.

Reply:
Gotcha, thanks for the clarification. 👌